trappy/plotter/ILinePlotGen.py

```diff
@@ -34,6 +34,24 @@
 
 from IPython.display import display, HTML
 
+def df_to_dygraph(data_frame):
+    """Helper function to convert a :mod:`pandas.DataFrame` to
+    dygraph data
+
+    :param data_frame: The DataFrame to be converted
+    :type data_frame: :mod:`pandas.DataFrame`
+    """
+    result = {}
+
+    values = data_frame.values.tolist()
+    data = [[x] for x in data_frame.index.tolist()]
```

Comment:
We should have kept the `data_dict` and have done this with dicts instead.  It would have been simpler and probably faster.  Anyway, let's merge it as it is, otherwise we're going to be spinning this series forever.